### DISCUSSION TITLE
Welcome to laser-level-webcam Discussions!

### DISCUSSION THREAD

**Discussion by bhowiebkr** [bhowiebkr/laser-level-webcam#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by miked243** [comment]
Hi, I'm mike. formally a mechanical engineer and currently getting my own machine shop up and running.
I came across your project while looking for an affordable way to quantify the lapping of my surface plate. I'm supper glad you did this as my coding skills are more like a toddler playing monkey see monkey do, so thank you so very much. I plan to add my build here and maybe youtube once I get everything done, I think it would help a lot with the hardware side of this.
